cime_config/buildnml

```diff
@@ -223,6 +228,11 @@ def buildnml(case, caseroot, compname):
         if pg_blom.get_value('inid14c') == 'UNSET':
             pg_blom.set_value('inid14c', value='')
 
+        if pg_blom.get_value('use_cisonew') == ".true.":
+            if pg_blom.bet_value("use_SEDBYPASS") == ".false.":
+                expect(False,
+                       "HAMOCC C-isotopes currently not supported in the sediment module. Set use_SEDBYPASS to .true.")
```

Comment:
This should be removed now, since C-isotopes with sediment is technically supported now.